frontend/src/actions/service.js

```diff
@@ -1,6 +1,13 @@
 import axios from "axios";
 
+const path = "http://localhost:4000/api/service";

```

Comment:
better to have this path in a single config file. From where it can be changed in a single go.